src/assets/Generator.Shared/StringRequestContent.cs

```diff
@@ -19,7 +19,11 @@ public StringRequestContent(string value)
 
         public override async Task WriteToAsync(Stream stream, CancellationToken cancellation)
         {
+#if NET6_0_OR_GREATER
+            await stream.WriteAsync(_bytes.AsSpan(0, _bytes.Length), cancellation).ConfigureAwait(false);
```

Comment:
I am not sure if this overload of WriteAsync exists in Stream for all of the TFMs used in the .NET Azure SDK repo so I conditioned it like this.

Reply:
It looks like it does exist in TFM’s that implement .NET Standard 2.1.
![2FD1335B-29A1-4ADB-B86F-6CE0415E9D8B](https://user-images.githubusercontent.com/15173749/169687168-37f0c6cc-5709-47c8-b1b6-db61de4c6184.png)